subsys/bluetooth/controller/hci/hci_driver.c

```diff
@@ -70,6 +74,31 @@ static sys_slist_t hbuf_pend;
 static s32_t hbuf_count;
 #endif
 
+static struct net_buf *process_prio_evt(struct node_rx_pdu *node_rx)
+{
+#if defined(CONFIG_BT_CONN)
+	struct pdu_data *pdu_data = PDU_DATA(node_rx);
+	struct net_buf *buf;
+	u16_t handle;
+
+	/* Avoid using hci_get_class() to speed things up */
```

Comment:
This argument seems strange. We call hci_get_class just before calling this function. so we haven't avoided it.
Is the optimization that we only call it once? If so I would remove the comment here and write a comment about that where hci_get_class was called and cached.

Reply:
Sorry, this is a stale comment! I will remove it

Reply:
Fixed